_config.yml

```diff
@@ -8,9 +8,10 @@
 # Site settings
 title: Astral Dynamics
 email: contact@astraldynamics.co.uk
-description: >
-  We are a forward thinking tech company with a focus on enhancing culture
-  with technology.
+description: > 
+  Astral Dynamics is an Innovation and Rapid Proto-typing agency. We are a multi- disciplinary 
```

Comment:
Innovation and multi-disciplinary are buzzwords. Innovation is a quality that should be inherent in the company's success, not a label they attach to themselves to sound better.

Likewise, most teams are multi-disciplinary. Especially in the startup world. All we need to do is explain what we do. The second part of the second sentence is better, but you've repeated rapid prototype _and_ you've written it differently. Proto-typing vs prototype.

This description is meant to be a snappy sales pitch. It should be free of jargon and distilled to the absolute core of what makes Astral Dynamics an important player.